### PR TITLE
chore: add volunteer anand007s

### PR DESCRIPTION
**Pull request by anand007s**

### CODE REVIEW

File: data/users/anand007s.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "bio": "Developer",
+  "image": "https://twitter.com/anand01twt/photo",
+  "categories": [
+    {
+      "slug": "Web-developer",
+      "topics": ["Linux","Markdonw","Html","Css","Javascript","ReactJs","Java","SpringBoot"]
+    }
+  ],
+  "contacts": {
+    "twitter": "@anand01twt"
```
Comment (codewithvoid) [inline data/users/anand007s.json:12]: @ is not required here

File: data/users/anand007s.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Anand",
+  "bio": "Developer",
+  "image": "https://twitter.com/anand01twt/photo",
+  "categories": [
+    {
+      "slug": "Web-developer",
```
Comment (codewithvoid) [inline data/users/anand007s.json:7]: slug is specified in snake case
```json
"slug": "web-development"
```

File: data/users/anand007s.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Anand",
+  "bio": "Developer",
+  "image": "https://twitter.com/anand01twt/photo",
```
Comment (codewithvoid) [inline data/users/anand007s.json:4]: use this image url
```json
"image": "https://pbs.twimg.com/profile_images/1621920209446780929/EqXUbX04_400x400.jpg"
```

File: data/users/anand007s.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  "categories": [
+    {
+      "slug": "Web-developer",
+      "topics": ["Linux","Markdonw","Html","Css","Javascript","ReactJs","Java","SpringBoot"]
```
Comment (codewithvoid) [inline data/users/anand007s.json:8]: limit topics to max 2-3

File: data/users/anand007s.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Anand",
+  "bio": "Developer",
```
Comment (codewithvoid) [inline data/users/anand007s.json:3]: optional: `Web Developer`

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering @anand007s. Added some comments.

**Review by anand007s — COMMENTED** [review]
Changes done as per above comments.

File: data/users/anand007s.json (hunk 1 above)
Comment (anand007s) [reply data/users/anand007s.json]: Done

File: data/users/anand007s.json (hunk 2 above)
Comment (anand007s) [reply data/users/anand007s.json]: Done

File: data/users/anand007s.json (hunk 3 above)
Comment (anand007s) [reply data/users/anand007s.json]: Done

File: data/users/anand007s.json (hunk 5 above)
Comment (anand007s) [reply data/users/anand007s.json]: Done

**Comment by anand007s** [comment]
Changes are Done.

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes. Welcome!

**Pull request merged**